### ISSUE TITLE
Keyboard help button highlighted upon loading sim

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/resistance-in-a-wire#139]
**Test device:**

Hanson

**Operating System:**

Windows 10

**Browser:**

Firefox ESR 52.6.0

**Problem description:**

@jessegreenberg I noticed that upon loading the sim with NVDA running, the keyboard help button is highlighted in the usual pink/purple color to indicate keyboard navigation. I didn't press any buttons or click anything with the mouse. Related to phetsims/QA/issues/108.

EDIT: This can happen with anything you can tab to, not just the keyboard help button.

**Steps to reproduce:**

1. Start NVDA.
2. Tab to something.
3. Reload the sim. (The sim should load with the thing you tabbed to highlighted.)

**Troubleshooting information (do not edit):**

Name: ‪Resistance in a Wire‬
URL: https://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.5.0-rc.1/phet/resistance-in-a-wire_all_phet.html
Version: 1.5.0-rc.1 2018-03-09 23:10:33 UTC
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:52.0) Gecko/20100101 Firefox/52.0
Language: en-US
Window: 1600x764
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"HEAD"},"axon":{"sha":"cc053b4d","branch":"HEAD"},"brand":{"sha":"89d28f63","branch":"HEAD"},"chipper":{"sha":"54f17be2","branch":"HEAD"},"dot":{"sha":"0dd6ee8e","branch":"HEAD"},"joist":{"sha":"9ca4cd0a","branch":"HEAD"},"kite":{"sha":"b6071478","branch":"HEAD"},"phet-core":{"sha":"f35ff65e","branch":"HEAD"},"phet-io":{"sha":"d54be499","branch":"HEAD"},"phet-io-website":{"sha":"28284828","branch":"HEAD"},"phet-io-wrapper-classroom-activity":{"sha":"c84e3046","branch":"HEAD"},"phet-io-wrapper-lab-book":{"sha":"ebf7c7dc","branch":"HEAD"},"phet-io-wrappers":{"sha":"ce57c3e2","branch":"HEAD"},"phetcommon":{"sha":"a5a7478c","branch":"HEAD"},"query-string-machine":{"sha":"485e174e","branch":"HEAD"},"resistance-in-a-wire":{"sha":"39b3ad26","branch":"HEAD"},"scenery":{"sha":"ddef8aa5","branch":"HEAD"},"scenery-phet":{"sha":"42b0cff2","branch":"HEAD"},"sherpa":{"sha":"9c6ec322","branch":"HEAD"},"sun":{"sha":"6a7c923a","branch":"HEAD"},"tandem":{"sha":"7f499d9e","branch":"HEAD"}}

**Comment by lmulhall-phet** [comment]
EDIT: In order for this to work, you need to reload the sim. Seems to be isolated to FF ESR (can't reproduce in Chrome). Checking to see if it happens with FF Quantum.

**Comment by jessegreenberg** [comment]
Sorry  @lmulhall-phet I should have let you know about this - this is a screen reader feature not a but. Screen readers (especially NVDA) will try to remember  the last thing that had focus and restore it on page and return  focus to it on page load.  I will make a note of it in the testing template.

**Comment by lmulhall-phet** [comment]
Ah, I was wondering if it was a feature... It seems too convenient to be a bug. Thanks @jessegreenberg.

**Comment by jessegreenberg** [comment]
I made a note for next time. Closing this issue, thanks  @lmulhall-phet.
